Changelog.md

```diff
@@ -16,6 +16,7 @@ Compiler Features:
 
 Bugfixes:
  * Name Resolver: Fix that when importing an aliased symbol using ``import {AliasedName} from "a.sol"`` it would use the original name of the symbol and not the aliased one.
+ * Parser: Properly check for multiple SPDX license indentifiers and validate them.
```

Comment:
```suggestion
 * Parser: Properly check for multiple SPDX license identifiers and validate them.
```

Reply:
Also maybe it would be good to clarify a bit:
```suggestion
 * Parser: Properly check for multiple SPDX license identifiers next to each other and validate them.
```